### ISSUE TITLE
Disconnecting Issur with SR300 

### ISSUE THREAD

**Issue by ckim337** [teknotus/depthview#14]
Hello, I am currently using realsense SR300 in Ubuntu 14.04. 
(model name : Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz) -using USB 3.0 

After connecting the SR300 camera to the computer, computer cannot detect the USB from SR300. 
Error saying that "cannot detect video4linux". In my understanding, this might be the power problem. So we tried with powered hub USB that has 4 Port USB 3.0 Hub with 5V/2.5A power adapter, it did not work neither. 

-Can anyone know the exact power draw of realsense SR300? 
-If someone already have used the powered hub for SR300 or F200, please let me know the information about the that product. Thank you !

**Comment by vcazan** [comment]
Have you installed v4l with apt-get?

We have had good luck with this hub: https://www.amazon.ca/D-Link-4-Port-SuperSpeed-USB-DUB-1340/dp/B005AS337Y

**Comment by vcazan** [comment]
also does the device show up in /dev/video*

**Comment by ckim337** [comment]
@vcazan It worked before for a day. When I tried again after, started to showing errors. Was the powered hub for SR300 or F200?

**Comment by vcazan** [comment]
have both and works well for both

**Comment by ckim337** [comment]
Thank you ! let me order the powered hub and try it again with that

**Comment by teknotus** [comment]
Often different ports on a computer provide different amounts of power. Did
you try all of the ports?

**Comment by ckim337** [comment]
@teknotus Yes, I have tried all the 3.0 USB ports in 4 different computers. Didn't work...
